kernel/base/src/main/java/com/twosigma/beakerx/BeakerxClientManager.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
rename BeakerxClientManager.java to BeakerXClientManager.java

Reply:
done